src/testRunner/unittests/tsserver/plugins.ts

```diff
@@ -194,7 +194,8 @@ describe("unittests:: tsserver:: plugins overriding getSupportedCodeFixes", () =
                                 case "/b.ts":
                                     return ["b"];
                                 default:
-                                    return info.languageService.getSupportedCodeFixes(fileName);
+                                    // Make this stable list of single item so we dont have to update the baseline for every additional error
+                                    return [info.languageService.getSupportedCodeFixes(fileName)[0]];

```

Comment:
If we happen to change what supported code fix comes first, will this also change? Maybe that's unlikely...

Reply:
yep thats probably a rare one compared to now when we have to update this everytime.